src/controllers/apartment.ts

```diff
@@ -578,9 +582,9 @@ export const getCreateApartment = (req: Request, res: Response) => {
  * Create landlord's apartment.
  */
 export const postCreateApartment = async (req: Request, res: Response, next: NextFunction) => {
-    await check("apartmentNumber", "Apartment number must be a number (without commas).").exists().isNumeric().run(req);
-    await check("numBedrooms", "Number of bedrooms must be a number (without commas).").exists().isNumeric().run(req);
-    await check("numBathrooms", "Number of bathrooms must be a number (without commas).").exists().isNumeric().run(req);
+    await check("apartmentNumber", "Apartment number must be a number (e.g., 1200, 1,200, or $1200).").exists().custom((value: string) => validatePrice(value)).run(req);
```

Comment:
It doesn't make sense for the apartment number of a condo to be a price. Like it shouldn't allow people to say apartmentNumber = $1,200.